Build/PackageFiles/ProjectTemplates-Isolated.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Are you planning to add F# project template as well?

Reply:
For now, we aren't planning on. We may later. But this release won't include them.